### DISCUSSION TITLE
Distribution of points to base stats during character creation.

### DISCUSSION THREAD

**Discussion by fugakuar** [AscensionGameDev/Intersect-Engine#1491] [Ideas]
**Is your feature request related to a problem? Please describe.**
Not really, but it's a really nice addition to add distribution points for base stats. In most MMOs they have the option that when creating a character you add a certain amount of points to the statistics, before the creation of the character.

**Describe the solution you'd like**
Create an option to add or remove this function in some sector of the server config. In the character creation you can add the list of base stats where the user can add or remove points to his stats before creation by means of arrows.

**Additional context**
N/A

**Comment by WeylonSantana** [comment]
To me, it makes perfect sense to add this IF and ONLY IF the class editor had start points to distribute (obviously), but as far as I know, the character selection window only has the basic character information, so I don't know what how much would have to be rewritten for this to work

**Comment by da-rulez** [comment]
I'm no fan of cheap workarounds but you could also just start characters off with X amount of stat points right? 

If there is no event command to give stat points, that one should be in hah
